### ISSUE TITLE
Migrations fail with escaped single quote characters from mysqldump in 4.0

### ISSUE THREAD

**Issue by gludington** [flyway/flyway#1418]
This appears the same behavior in https://github.com/flyway/flyway/issues/382 - and these scripts worked with the flyway 3.x series; however, when we moved to flyway 4.0, the issue is present.
###### What version of Flyway are you using?

4.0 
###### Which client are you using? (Command-line, Java API, Maven plugin, Gradle plugin, SBT plugin, ANT tasks)

Maven plugin
###### What database are you using (type & version)?

MySql to deploy, h2 for testing
###### What operating system are you using?

Linux, OSX
###### What did you do?

(Please include the content causing the issue, any relevant configuration settings, and the command you ran)
Ran our test suite using mvn test, which begins by bootstrapping an h2 test database from sql scripts, one of which had originally been taken from mysqldump
###### What did you expect to see?

The migrations that worked int he 3.1.x series to be applied in the 4.0 series
###### What did you see instead?

Migration would fail with a sql syntax error due to the escaped single quote \'

In other words, lines like this threw an error in 4.0:

INSERT INTO `tbl` (`id`, `documentation`) VALUES (10,'i\'m not sure this works');
but would succeed with this:
INSERT INTO `tbl` (`id`, `documentation`) VALUES (10,'i''m not sure this works');

The first format would work using the older version of flyway.

**Comment by axelfontaine** [comment]
I am afraid that first line isn't valid SQL. Please open a bug report with MySQL to fix mysqldump.
